### PR TITLE
css: prettified /help page

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- optimized for mobile
- gave breathing space for FAQ and added basic animation (not visible in the gif )
- changed fonts to OpenSans
- refactored skeleton

help

##### Checklist

- [ ] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

- css

##### Refers/Fixes
- https://github.com/gitcoinco/web/pull/333 // The slack image in the footer is replaced with an icon here
- https://github.com/gitcoinco/web/pull/323 // Required font used here

### CODE REVIEW

File: app/assets/v2/css/help.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
+body {
+  font-family: "OpenSans";
```
Comment (owocki) [inline app/assets/v2/css/help.css]: we need to consolidate either around Futura or OpenSans... cc @pixelantdesign

Comment (thelostone-mc) [reply app/assets/v2/css/help.css]: refer why the change is being made : https://github.com/gitcoinco/web/issues/133#issuecomment-352750331

I couldn't find the conversation which algae12 refers on slack ( due to the free versions memory limited 😓  )

**Comment by PixelantDesign** [comment]
Hello! I think we should stick with Futura for the time being. Would it be possible to set the tracking to zero and add hover states #3E00FF to the topics on the help site?

**Comment by thelostone-mc** [comment]
@owocki Bugged @PixelantDesign offline and got things changed. review + merge before I have to resolve more conflicts ? 

the on hover is also added! ^_^

screenshot-2018-1-31 gitcoin push open source repos forward

**Comment by owocki** [comment]
lgtm; @mbeacom ?

File: app/retail/templates/help.html [hunk 2]
```diff
@@ -22,21 +22,21 @@
 <head>
     {% include 'shared/head.html' %}
     {% include 'shared/cards.html' %}
+    <link rel="stylesheet" href="/static/v2/css/help.css" />
```
Comment (mbeacom) [inline app/retail/templates/help.html]: You'll want to update this to `<link rel="stylesheet" href="{% static "v2/css/help.css" %}">`

File: app/retail/templates/help.html [hunk 3]
```diff
@@ -76,52 +76,56 @@ <h6 style="" >Check out the Gitcoin Case Study</h6>
...
-                        <div class='faq_parent mb-4'>
-                            <h5>{{item.q}}</h5>
+                      <li>  <div class='faq_parent mb-4'>
+                            <p class="question">{{item.q}}</p>
                             <div class='hidden answer'>{{item.a|safe}}
                         </div>
-                    </div>
+                    </div> </li>
                     {% endfor %}
+                  </ul>
```
Comment (mbeacom) [inline app/retail/templates/help.html]: Can you fix the indentation for this block?

**Comment by thelostone-mc** [comment]
@mbeacom Done!

**Pull request merged**